src/AspNet.Security.OAuth.VkId/VkIdAuthenticationExtensions.cs

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace AspNet.Security.OAuth.VkId;
+
+/// <summary>
+/// Extension methods to add VK ID authentication capabilities to an HTTP application pipeline.
+/// </summary>
+public static class VkIdAuthenticationExtensions
+{
+    /// <summary>
+    /// Adds <see cref="VkIdAuthenticationHandler"/> to the specified
+    /// <see cref="AuthenticationBuilder"/>, which enables Vkontakte authentication capabilities.
```

Comment:
Should all these references to "Vkontakte" be "VK ID"?

Reply:
Not sure 😅
VK ID = OAuth2.1 vkontakte implementation

Reply:
The reason I ask is because it's referred to as "VK ID" elsewhere in the XML documentation comments.

Reply:
Yea, changed it to VK ID to not confuse people